nilearn/plotting/img_plotting.py

```diff
@@ -815,6 +834,32 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
     return display
 
 
+def _cmap_discretize(cmap, N):
+    """Return a discrete colormap from the continuous colormap cmap.
+
+        cmap: colormap instance, eg. cm.jet.
+        N: number of colors.
+
+    Example
+        x = resize(arange(100), (5,100))
+        djet = cmap_discretize(cm.jet, 5)
+        imshow(x, cmap=djet)
```

Comment:
Could you format this according to the numpy doc standard: a Parameters section, and Example as a section.

The reason that I ask this, is that the docstring is invalid currently.

Also, the examples should be doctests.